components/script/dom/htmlimageelement.rs

```diff
@@ -3,6 +3,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use app_units::{Au, AU_PER_PX};
+use core::ops::Deref;
```

Comment:
we are using this at line: 802:` let unparsed_sizes = input.deref().split(',').collect::<Vec<_>>();`